Automattic/sensei
# Update section headings in Landing Page and Course List patterns

Fixes #6213.

Updates the `h5` tags to `h2` tags for the section headers in the Landing Page and Course List patterns. Also updates their font size to Medium.

### Testing instructions

- Add the Landing Page pattern to a page.
- Ensure the "Course List", "Blog Posts" and "What Students Say" section headers are `h2` and the font size is medium.
- Repeat with the Landing Page - Grid, Course List and Course List - Grid patterns.
